packages/swing-store/src/swingStore.js

```diff
@@ -826,7 +826,6 @@ function makeSwingStore(dirPath, forceReset, options = {}) {
    */
   async function close() {
     db || Fail`db not initialized`;
-    commit();
     db.close();
```

Comment:
Out of curiosity, why not explicitly rollback before closing?

```suggestion
    if (db.inTransaction) {
      db.prepare('ROLLBACK').run();
    }
    db.close();
```

Reply:
Answering my own question from the docs:
> But a transaction will also ROLLBACK if the database is closed

Reply:
I tried that and I got SQLITE_ERROR, I wasn't able to trivially find out why.